app/src/main/res/values/strings.xml

```diff
@@ -972,6 +972,41 @@
     <string name="split_tall_images">Split tall images</string>
     <string name="split_tall_images_summary">Improves reader performance</string>
 
+    <!-- Stats -->
+    <string name="stats">Stats</string>
+    <string name="statistics">Statistics</string>
+    <string name="statistics_details">Statistics Details</string>
+    <string name="search_statistics">Search statistics</string>
+    <string name="stat_">Stat: %1$s</string>
+    <string name="total_manga">Total Manga</string>
+    <string name="chapters_read">Chapters Read</string>
+    <string name="mean_score">Mean Score</string>
+    <string name="manga_tracked">Manga Tracked</string>
+    <string name="chapters_downloaded">Chapters Downloaded</string>
+    <string name="tags">Tags</string>
+    <string name="manga_local">Manga Local</string>
+    <string name="global_update_manga">Global Update Manga</string>
+    <string name="trackers">Trackers</string>
+    <string name="read_duration">Read Duration</string>
+    <string name="manga_status_distribution">Status Distribution</string>
+    <string name="manga_score_distribution">Score Distribution</string>
+    <string name="view_detailed_statistics">View Detailed Statistics</string>
+    <string name="count">Count</string>
+    <string name="not_rated">Not Rated</string>
+    <string name="source">Source</string>
+    <string name="serie_type">Serie Type</string>
```

Comment:
this should be series type right? series type already exists

Reply:
I think so too. Serie isn't an English word afaik.

Reply:
Oh yeah my bad, both words exist in my primary language (french)